recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - cf_units >=2
     - cftime
     - dask >=0.17.1
-    - numpy >=1.14
+    - numpy
```

Comment:
If `iris` still have a `numpy` build dependency for the extensions this should follow

https://conda-forge.org/docs/meta.html#building-against-numpy

Note that the global [`numpy` pinning is `1.9`](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L354) on Linux and OS X and `1.11` on Windows. If you need different version you can specify that on `host`.

If you do not want to move this recipe to the new format just yet (the bot is doing that in the "right order") you can just use the old `numpy` pinning scheme:

```yaml
  build:
    - numpy 1.11.*
  run:
    - numpy >=1.11
```

Change the minimum version accordingly.

However, if `iris` no longer have a build dependency on `numpy` for the extensions, you can drop the pinning on `run` too like you just did for build.

Reply:
Answering my own question I don't think that is true since `mo_pack` was removed. Let's merge this one. Thanks @bjlittle!